finish/system/Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM open-liberty
 
-ADD --chown=1001:0 target/system.tar.gz /opt/ol
+COPY src/main/liberty/config /config/
```

Comment:
May still need to specify the permission when copying? ie, `COPY --chown=1001:0  server.xml /config/` as in the https://github.com/OpenLiberty/ci.docker#building-an-application-image doc.

Reply:
@MaiHameed is to find out more about the above. Pls add updates here.

Reply:
@NimG98 The latest update is to add the flag.

Reply:
Latest changes look good.